setup.py

```diff
@@ -46,12 +59,9 @@
                 "h5py>=2.7",
                 "pygmo>=2.7",
             ],
-            "probabilistic": [
-                "pymc3>=3.8",
-                "theano>=1.0",
-            ],
+            "probabilistic": ["pymc3>=3.8", "theano>=1.0",],
```

Comment:
My only nitpick is that I would prefer to keep unrelated formatting changes out of functional commits (after the release would probably be a good time to continue with #78). But that's not terribly important.

Otherwise looks good to me :)

Reply:
I agree that this can obfuscate the true changes of the commit. I should turn off my auto-formatting on save.